saylua/modules/messages/models/db.py

```diff
@@ -1,6 +1,45 @@
 from google.appengine.ext import ndb
 
 import datetime
+from saylua import db
+
+
+class Conversation(db.Model):
+    __tablename__ = "conversations"
+
+    id = db.Column(db.Integer, primary_key=True)
+    title = db.Column(db.String(256))
+    author = db.Column(db.Integer, db.ForeignKey('users.id'))
+
+
+class ConversationMember(db.Model):
+    __tablename__ = "conversation_members"
+
+    conversation_id = db.Column(db.Integer, db.ForeignKey('conversations.id'), primary_key=True)
+    user_id = db.Column(db.Integer, db.ForeignKey('users.id'), primary_key=True)
+    unread = db.Column(db.Boolean, default=False)
+
```

Comment:
Given the way messages are used, I think it would be best to denormalize the title from Conversation into this model, so we don't have to do so many join queries everywhere when looking up someone's messages. 

Also, add a last updated time for this field for similar reasons. Since messages are displayed on every page, it's worth it to have a little bit of denormalization, I think.